themes/src/main/resources-community/theme/base/login/messages/messages_de.properties

```diff
@@ -208,6 +208,8 @@ usernameExistsMessage=Benutzername existiert bereits.
 emailExistsMessage=E-Mail existiert bereits.
 
 federatedIdentityExistsMessage=Ein Benutzer mit {0} {1} existiert bereits. Bitte melden Sie sich an der Benutzerkontoverwaltung an um den Benutzer zu verknüpfen.
+federatedIdentityUnavailableMessage=Der mit dem Identitätsanbieter {1} authentifizierte Benutzer {0} ist nicht vorhanden. Bitte wenden Sie sich an Ihren Administrator.
+federatedIdentityUnmatchedEssentialClaimMessage=Das von dem Identitätsanbieter ausgestellte ID-Token entspricht nicht der konfigurierten mind. Anforderrung. Bitte kontaktieren Sie Ihren Administrator.
```

Comment:
```suggestion
federatedIdentityUnmatchedEssentialClaimMessage=Dem vom Identitätsanbieter ausgestellten ID-Token fehlen die passenden Claims. Bitte kontaktieren Sie Ihren Administrator.
```

Reply:
Thanks for your feedback - i have adopted your suggestions